R/chartutils.R

```diff
@@ -1223,9 +1223,13 @@ setTicks <- function(minimum, maximum, distance, reversed = FALSE,
     {
         is.bar <- grepl("Bar", type) && !grepl("Stacked", type)
         if (is.null(minimum))
-            minimum <- min(0, min(data, na.rm = TRUE))
+            minimum <- min(data, na.rm = TRUE)
         if (is.null(maximum))
             maximum <- max(data, na.rm = TRUE)
+        if (all(data > 0, na.rm = TRUE))
```

Comment:
So if the minimum is manually set, we still override it when all data is positive?

Reply:
But we didn't override the minimum and maximum before, so wouldn't this change affect existing charts?

Reply:
I've modified the fix so it should only affect the case where there is a single value. To see the change, I added some tests and the changes caused by this PR are here: [3333ba1](https://github.com/Displayr/flipStandardCharts/pull/117/commits/3333ba1521a920ef18bd25244bedf2c39ca7835e)